apps/ideaspace/src/components/modules/IdeasBetaFeedbackModal/IdeasBetaFeedbackModal.js

```diff
@@ -45,36 +48,40 @@ const IdeasBetaFeedbackModal = () => {
     setShowModal(false);
   }
 
-  Modal.setAppElement('#__next');
+  useEffect(() => {
+    Modal.setAppElement('#__next');
+  }, [])
 
   return (
     <div>
-      <button className="open-button" onClick={openModal}>
-        <img src={messageIcon} />
+      <button className={style['open-button']} onClick={openModal}>
+        <img src={messageIcon.src} />
       </button>
       <Modal
         isOpen={showModal}
         onRequestClose={closeModal}
-        style={ModalCustomStyles}
+        // style={ModalCustomStyles}
```

Comment:
can this be deleted?

Reply:
@dbradham these changes are from your end, are you sure your team didn't comment this just for a bit then uncomment?

Reply:
yes, we used css modules instead of inline styles